### PR TITLE
Fixes #31067 - improve error message for ESP chainload

### PR DESCRIPTION
**Pull request by lzap**
Spent quite some today figuring why a VM can't chainboot. One damn checkbox...

### CODE REVIEW

**Comment by lzap** [comment]
Fixed snapshot tests.

**Comment by lzap** [comment]
This is also ready, let's get this in for 2.3 please.

File: app/views/unattended/provisioning_templates/snippet/pxegrub2_chainload.erb [hunk 1]
```diff
@@ -43,8 +43,15 @@ menuentry 'Chainload Grub2 EFI from ESP' --id local_chain_hd0 {
...
-  echo the list of known filepaths for probing. Contents of /EFI directory:
-  ls ($chroot)/EFI
+  echo the list of known filepaths for probing. Available devices are:
+  echo
+  ls
+  echo
+  If you can't see the HDD, make sure the drive is marked as bootable in EFI and
+  not hidden. Boot order must be the following:
+  1) NETWORK
+  2) HDD
```
Comment (tbrisker) [inline app/views/unattended/provisioning_templates/snippet/pxegrub2_chainload.erb]: I'm not very familiar with pxegrub syntax, but shouldn't these be?
```suggestion
  echo If you can't see the HDD, make sure the drive is marked as bootable in EFI and
  echo not hidden. Boot order must be the following:
  echo 1) NETWORK
  echo 2) HDD
```

Comment (lzap) [reply app/views/unattended/provisioning_templates/snippet/pxegrub2_chainload.erb]: Forgot to repeat `:normal Iecho ` in Vim on the other lines :)

**Comment by lzap** [comment]
Thanks, rebasing.

**Review by tbrisker — APPROVED** [review]
Thanks @lzap !

**Pull request merged**